packages/lesswrong/lib/voting/reactions.tsx

```diff
@@ -200,14 +212,14 @@ export const namesAttachedReactions: NamesAttachedReactionType[] = [
     name: "coveredAlready",
     label: "I already addressed this",
     searchTerms: ["check", "already", "covered", "addressed"],
-    svg: "/reactionImages/nounproject/checkedbox.svg",
-    filter: {opacity: 0.6},
+    svg: "/reactionImages/nounproject/noun-mail-checkmark-5316519.svg",
+    filter: {opacity: 0.6, scale: 1.2, translateY: 2},
     description: "I covered this in my post and/or comments.",
     deprecated:false,
   },
   {
     name: "unnecessarily-combative",
-    label: "Unnecessarily Combative",
+    label: "Too Combative?",
```

Comment:
I think the "unnecessarily" does important work in not saying that combativeness is not generically something we are against.